python/sglang/srt/server.py

```diff
@@ -172,6 +172,19 @@ async def stop_profile():
     )
 
 
+@app.get("/get_memory_pool_size")
+@app.post("/get_memory_pool_size")
```

Comment:
`@app.api_route("/get_memory_pool_size", methods=["GET", "POST"])`

This might be cleaner